azure-pipelines.yml

```diff
@@ -64,6 +47,20 @@ jobs:
       filename: script\download-spark-distros.cmd
       arguments: $(Build.BinariesDirectory)
 
+  - script: build.cmd -pack
+              -c $(buildConfiguration)
+              $(_OfficialBuildIdArgs)
+              /p:PublishSparkWorker=true
+              /p:SparkWorkerPublishDir=$(Build.ArtifactStagingDirectory)\Microsoft.Spark.Worker
+    displayName: '.NET build'
+
+  - task: DotNetCoreCLI@2
+    displayName: '.NET unit tests'
+    inputs:
+      command: test
```

Comment:
Does `build.cmd -test /p:ProjectToBuild=**\*UnitTest\*.csproj` work?

Reply:
No, unfortunately it doesn't. We can't run tests using arcade's test runner yet because it doesn't support coverlet, so we would lose code coverage. I started an internal discussion on the Arcade Working Group on this topic, but didn't get very far. Basically https://github.com/dotnet/arcade/issues/1946 is the blocking issue.

Reply:
Ok, cool. Once that gets resolved we can think of updating this.